core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -227,4 +227,12 @@ abstract contract FeePayer is Testable {
         rawCollateral.rawValue = rawCollateral.add(adjustedCollateral).rawValue;
         addedCollateral = _getCollateral(rawCollateral).sub(initialBalance);
     }
+
+    // Scale the CumulativeFeeMultiplier by a given amount based on the current PFC value.
```

Comment:
nit: Change "CumulativeFeeMultiplier" to "`cumulativeFeeMultiplier`" so you're using the exact name of the contract state variable. Note the small ticks.

Reply:
I think this comment could also be clearer: "Scale down the `cumulativeFeeMultiplier` by the ratio of fees paid to the current profit from corruption"